plugins/voltageclamp/relacs.cfg

```diff
@@ -1,17 +1,19 @@
 *Settings
   Plugins:
-      pluginpathes    : [ base*, calibration*, ephys*, patchclamp*, voltageclamp*, comedi*, rtaicomedi*, dynclampsim ]
+      pluginpathes    : [ base*, base*, calibration*, ephys*, patchclamp*, voltageclamp*, comedi*, rtaicomedi*, dynclampsim ]
       pluginhelppathes: ~
-      controlplugin   : [ Session, AmplifierControl, SpectrumAnalyzer ]
+      controlplugin   : [ Session, Session, AmplifierControl, SpectrumAnalyzer, PeriodHistogram ]
       modelplugin     : NeuronModels
   Pathes:
-      pathformat : %04Y-%02m-%02d-%a2a
-      defaultpath: dry/
+      pathformat : /home/sonnenberg/Pycharm/PycharmProjects/ModelProtocols/NixTest/%04Y-%02m-%02d-%a2a
```

Comment:
No absolute path! ./relacslocal should work, therefore simply set
pathformat : %04Y-%02m-%02d-%a2a